src/Umbraco.Web/Umbraco.Web.csproj

```diff
@@ -257,7 +257,16 @@
     <Compile Include="PropertyEditors\BlockListConfigurationEditor.cs" />
     <Compile Include="PropertyEditors\BlockListPropertyEditor.cs" />
     <Compile Include="Compose\NestedContentPropertyComposer.cs" />
+    <Compile Include="PropertyEditors\IFileExtensionConfigItem.cs" />
+    <Compile Include="PropertyEditors\FileExtensionConfigItem.cs" />
+    <Compile Include="PropertyEditors\IFileExtensionConfig.cs" />
     <Compile Include="PropertyEditors\ComplexEditorValidator.cs" />
+    <Compile Include="PropertyEditors\FileUploadConfiguration.cs" />
+    <Compile Include="PropertyEditors\FileUploadConfigurationEditor.cs" />
+    <Compile Include="PropertyEditors\IFileExtensionConfigItem.cs" />
```

Comment:
I'm getting a few "Ambiguous reference" errors as well, the project compiles just fine though, but I can see that "PropertyEditors\IFileExtensionConfigItem.cs" is added twice to the csproj, so that might be part of the reason at least

Reply:
I am not seeing any ambiguous references but I am using VS and not Rider. Let me know if its still a problem after the dupe in CSProj removed

Reply:
It seems to have gone away not that the dupe has been removed 👍